pkgs/misc/emulators/fceux/default.nix

```diff
@@ -1,11 +1,13 @@
-{stdenv, fetchurl, scons, zlib, SDL, lua5_1, pkgconfig}:
+{stdenv, fetchFromGitHub, scons, zlib, SDL, lua5_1, pkgconfig}:
 
 stdenv.mkDerivation {
-  name = "fceux-2.2.3";
+  name = "fceux-2.2.3-20200129";
 
-  src = fetchurl {
-    url = mirror://sourceforge/fceultra/Source%20Code/2.2.3%20src/fceux-2.2.3.src.tar.gz;
-    sha256 = "0gl2i3qdmcm7v9m5kpfz98w05d8m33990jiwka043ya7lflxvrjb";
+  src = fetchFromGitHub {
+    owner = "TASVideos";
+    repo = "fceux";
+    sha256 = "0gpz411dzfwx9mr34yi4zb1hphd5hha1nvwgzxki0sviwafca992";
+    rev = "fb8d46d9697cb24b0ebe79d84eedf282f69ab337";
```

Comment:
I'd usually swap these lines but it doesn't really matter (only FYI, the result doesn't change).

Reply:
Swapped.